instrumentation/jaxws/jws-1.1/javaagent/src/test/java/io/opentelemetry/test/WebServiceFromInterface.java

```diff
@@ -3,8 +3,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.javaagent.instrumentation.jaxws.jws.v1_1;
+package io.opentelemetry.test;
 
+import io.opentelemetry.javaagent.instrumentation.jaxws.jws.v1_1.WebServiceDefinitionInterface;
```

Comment:
Should we move `WebServiceDefinitionInterface` out of the javaagent package as well?